app/services/converter/base.rb

```diff
@@ -50,5 +50,17 @@ def send_mutation(mutation, token)
       Rails.logger.error mutation
       Rails.logger.error '#' * 30
     end
+
+    private
+
+      # in https://github.com/github/graphql-client/blob/v0.15.0/lib/graphql/client.rb#L123-L167
+      # there occur some errors when parsing strings with a certain regex, so we need to append
+      # a space after "..." in the matched string
+      def safe_parse(str)
+        str.gsub(/(\.\.\.)([a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)*)/) do
+          match = Regexp.last_match
+          match[1] + " " + match[2]
+        end
```

Comment:
this goes through every matched part in a big string and adds a space between "..." and "xyz".

example:

> hello ... world ...with multiple ...matches...in...this...string

becomes

> hello ... world ... with multiple ... matches... in... this... string

Reply:
Eine Kürzere Schreibweise wäre:

`str.gsub(/(\.\.\.)([a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)*)/, "#{$1} #{$2}")`

![Bildschirmfoto 2022-10-19 um 12 11 03](https://user-images.githubusercontent.com/90779/196663067-08408c1b-a809-4e94-9d74-6aec7ef7bb51.png)

Reply:
Nope, this would result in

> hello ... world multiple

On the other hand, rubocop changes your suggestion to `str.gsub(/(\.\.\.)([a-zA-Z0-9_]+(::[a-zA-Z0-9_]+)*)/, "#{Regexp.last_match(1)} #{Regexp.last_match(2)}")`.

So it is not very short and readable, beside, that it dies not work for all occurrences.

Reply:
ok, i guess that we will always have two groups because the regex only catches certain string combinations. but i will change `match[1] + " " + match[2]` to `"#{match[1]} #{match[2]}"` to avoid that errors you mention.

Reply:
fancy..... go for it